### ISSUE TITLE
Commit `.github/weekly-digest.yml` in a separate PR

### ISSUE THREAD

**Issue by aps120797** [probot/weekly-digest#17]
Instead of committing directly to master branch, opening a PR to add `.github/weekly-digest.yml` appears to be a better option.

**Comment by gr2m** [comment]
Agree

**Comment by Robdel12** [comment]
Absolutely agree. I didn't know there was a push that was going to happen. 

Also does there need to be a PR that is pushed if you're not changing any of the default settings yet?

**Comment by aps120797** [comment]
Yeah. Thanks for the feedback @Robdel12. Yes, a push directly to master would have been happened (This is as per the current version. I will be updating this by weekend, and will open a PR instead). 😄 

And yes, no PR is needed if you are happy with the default config.
The app automatically has some default configurations, which it will use if it doesn't find `.github/weekly-digest.yml` file in your repository.

**Comment by abhijeetps** [comment]
@steveoh , the bot will work only with _.github/weekly-digest.yml_.
Thank you for pointing that out!
I have updated the docs! 😄
